ssv/spectest/1 duties 30 attestation.json

```diff
@@ -1 +0,0 @@
UFPV6e","Signer":2},{"SharePubKey":"rB18mw38vhHez9rBWw56DkuqrPSOedI2ur5Y3TyVXN5yypXi42t0gAJ50Fc21UDa","Signer":3},{"SharePubKey":"rzXb9zreo+J1R+PZTobdrr0sy05CaNkuUpmnH+kaweVG60WnXOy20V9TjyVsS5gT","Signer":4}],"DomainType":[0,0,3,1],"FeeRecipientAddress":[83,89,83,181,166,4,0,116,148,140,241,133,234,167,210,171,189,102,128,143],"Graffiti":"AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="},"7":{"ValidatorIndex":"7","ValidatorPubKey":[182,187,88,11,153,123,142,193,156,228,163,244,158,119,162,72,61,109,237,99,201,77,188,41,237,100,195,41,155,134,67,66,192,22,209,140,218,3,24,20,87,187,137,33,251,51,251,174],"SharePubKey":"hy8TsbO0lG+9PU2KbMqdTXY/V94aA3wsRIGdP/y0r4W9L9k271qcZSVhofZ+wUNa","Committee":[{"SharePubKey":"hy8TsbO0lG+9PU2KbMqdTXY/V94aA3wsRIGdP/y0r4W9L9k271qcZSVhofZ+wUNa","Signer":1},{"SharePubKey":"iSX7BAe6tLRzcSO6fET+0ROh0Zpn5kHL5bjurjzMX0GAcbwbPW4Dgs0K2Seqbd6l","Signer":2},{"SharePubKey":"i3WHpdoPfs46AE402nPwbRBMRYwaEIbv5LJ6gUsKi8aE+SOAS38oW58pU6df50vx","Signer":3},{"SharePubKey":"pB9F73gvOW60/Gixv7J5my2z/EjwT/AiPvJUxKKPWv32O1cD3N1W6zH+ohqcsEHu","Signer":4}],"DomainType":[0,0,3,1],"FeeRecipientAddress":[83,89,83,181,166,4,0,116,148,140,241,133,234,167,210,171,189,102,128,143],"Graffiti":"AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="},"8":{"ValidatorIndex":"8","ValidatorPubKey":[167,117,88,132,107,70,234,157,189,65,79,154,233,54,96,233,69,222,41,138,248,51,233,57,16,245,243,248,24,78,114,140,225,240,79,143,207,182,102,183,214,74,250,35,103,161,78,27],"SharePubKey":"hm5qJnVguvfpbz6u8pY1IqViSJalAX5s5MNqfZkekbNAVdHgmze3Y3pgUWA/qPZI","Committee":[{"SharePubKey":"hm5qJnVguvfpbz6u8pY1IqViSJalAX5s5MNqfZkekbNAVdHgmze3Y3pgUWA/qPZI","Signer":1},{"SharePubKey":"t1PSZKmYRi5y/3pMO/K0sex4FE9WEvpyAREUvg3NgmjX1YzutlYRSPrpxN5vPKL2","Signer":2},{"SharePubKey":"jAMokVARRZzp9SsJlN3BjVwfBIgnxZH+JYukh/W5uEMfbfeY85DCCPZmoNTKqeZI","Signer":3},{"SharePubKey":"tHLM2nSnwMF9LKvZ1FGX9C7H+bgUrEgxcME/wg6hAOy2NqFkVE1q+Ts2B3mJ0LZx","Signer":4}],"DomainType":[0,0,3,1],"FeeRecipientAddress":[83,89,83,181,166,4,0,116,148,140,241,133,234,167,210,171,189,102,128,143],"Graffiti":"AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="},"9":{"ValidatorIndex":"9","ValidatorPubKey":[138,199,165,132,133,26,139,87,160,42,39,30,81,3,186,123,206,172,67,192,113,77,85,220,255,55,220,182,56,140,209,203,246,157,36,154,45,37,118,148,137,156,172,161,125,255,93,142],"SharePubKey":"qDTRWIIPJ0Gc9x7VxWx166vcoTh9IPy1grA9f1SjyJSGbrb3F9HcoM4aMa9YMQ6b","Committee":[{"SharePubKey":"qDTRWIIPJ0Gc9x7VxWx166vcoTh9IPy1grA9f1SjyJSGbrb3F9HcoM4aMa9YMQ6b","Signer":1},{"SharePubKey":"o76ZqnV4U2T299nWWiSLzTeBV1p+ZXOm/cab4NJLvfuB9ACf1p28GmnmQWwTrCXY","Signer":2},{"SharePubKey":"oHEcRyyO2KE+fLn1EjuhO1Hd68lHUYAvclkpDQOKtdxd3FWUxnbmT+hyq6DuJXjp","Signer":3},{"SharePubKey":"tmxtejLvezeTdX3FsBy5Cl216BGWw0ARl1/0r1ZdqgBSpfZBKOwh4qzTUIg3idqI","Signer":4}],"DomainType":[0,0,3,1],"FeeRecipientAddress":[83,89,83,181,166,4,0,116,148,140,241,133,234,167,210,171,189,102,128,143],"Graffiti":"AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="}}}
```

Comment:
what are all those files not in the `generated` folder

well at least they are deleted